input_parameters.conf

```diff
@@ -144,19 +144,17 @@ orbfitlksy:    5
 #------------------------------------
 # Phase closure correction parameters
 
-# large_dev_thr: closure threshold for each pixel in multiples of pi, e.g. 0.5 = pi/2, 1 = pi.
-# avg_ifg_err_thr: ifgs with more than this fraction of pixels above the closure threshold will be dropped entirely.
-# loops_thr_ifg: ifgs are dropped entirely if they do not participate in at least this many closure loops.
-# max_loop_length: closure loops with up to this many edges will be used.
-# max_loops_per_ifg: A closure loop will be discarded if all constituent ifgs in that loop have
-#                    contributed already to a number of loops equal to this parameter.
-# subtract_median: subtract median during closure checks (1 = yes; 0 = no)
-large_dev_thr:     0.5
-avg_ifg_err_thr:   0.05
-loops_thr_ifg:     2
-max_loop_length:   4
-max_loops_per_ifg: 2
-subtract_median:   1
+# closure_thr:         Closure threshold for each pixel in multiples of pi, e.g. 0.5 = pi/2, 1 = pi.
+# avg_ifg_err_thr:     Ifgs with more than this fraction of pixels above the closure threshold, on average, will be dropped entirely.
```

Comment:
Because of https://github.com/GeoscienceAustralia/PyRate/pull/343, could change to:

"Ifgs with more than this fraction of pixels above the closure threshold in all loops it participates in, will be dropped entirely"